### DISCUSSION TITLE
Pool switching problems!!!

### DISCUSSION THREAD

**Discussion by bandusz** [Chia-Network/chia-blockchain#17159] [Support]
Hello!

We haven't been able to change the pool for days, no matter how much we enter the address of the other pool, nothing happens!!!
Unfortunately, it doesn't even do anything for self-selection!!!!

We are currently having issues with Migpool

**Comment by BrandtH22** [comment]
Hello @bandusz , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the pool switching errors, likely what happened is the initial pool change attempt did not have an adequate fee for timely processing with the current high activity of the chain and the latter attempts were rejected from the mempools due to a conflict. 

What I would try is clicking the "delete unconfirmed transactions" option in the GUI then reattempting the pool swap with a reasonable fee. 
If this still does not work then I would stop chia, rename the wallet db at `~\.chia\mainnet\wallet\db\` , start chia and let it sync, then reattempt the pool swap with a reasonable fee.

Let us know if you run into any issues or questions throughout the process :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
